framework/PageForm/PageForm.tsx

```diff
@@ -215,11 +217,17 @@ export function PageFormGrid(props: {
 
 export function PageFormSubmitButton(props: { children: ReactNode; style?: CSSProperties }) {
   const { isSubmitting, errors } = useFormState();
+  const { clearErrors } = useForm();
   const { t } = useTranslation();
   const hasErrors = errors && Object.keys(errors).length > 0;
   return (
     <Tooltip content={t('Please fix errors')} trigger={hasErrors ? undefined : 'manual'}>
       <Button
+        onClick={() => {
```

Comment:
It was decided to clean Errors in case we have errors that not specific per field. Thus when the form is submitted the second time the error will be displayed again if remains the same.

Reply:
This is a great solution to clearing form-level errors.